src/installation/guides/fde.md

```diff
@@ -233,12 +195,35 @@ a new file at `/etc/dracut.conf.d/10-crypt.conf` with the following line:
 install_items+=" /boot/volume.key /etc/crypttab "
 ```
 
-Next, install the boot loader to the disk.
+Now we will configure our bootloader, see the instructions on [Installing
+GRUB](chroot.md#installing-grub) to install the correct GRUB package for your
+system, but do not install it on the disk just yet (with `grub-install`) as we
+will need to modify it's configuration before doing so.
```

Comment:
I prefer the old language... "Next, install..." just sounds better than "Now we will..."